app/views/govuk_publishing_components/components/_attachment.html.erb

```diff
@@ -42,7 +42,9 @@
                 tabindex: "-1",
                 "aria-hidden": true,
                 data: data_attributes do %>
-      <% if attachment.document? %>
+      <% if attachment.thumbnail_url %>
+        <img src="<%= attachment.thumbnail_url %>" alt="<%= attachment.title %>" />
```

Comment:
Have added it now @andysellick, thanks 👍

Reply:
I looked up what we do with the alt tag in Whitehall and we leave it blank: https://github.com/alphagov/whitehall/blob/b3e4ab486afb283383e5ce0fc076b5a6a797bb17/app/helpers/document_helper.rb#L83

This seems the correct action since we don't really need to describe it, given it's next to the title - but we [do always need an alt attribute](https://design-system.service.gov.uk/styles/images/#alt-text) and that having the title as the alt text is a bit misleading as it doesn't describe the image (it'd be more accurately described as something like "a preview image of the PDF document: ''<title>")

You could possibly do this more succinctly with an image_tag helper:

```suggestion
        <% image_tag(attachment.thumbnail_url, alt: "") %>
```

Reply:
Thank you @kevindew it makes sense 👍